### DISCUSSION TITLE
transfrom:scale自适应，坐标不准

### DISCUSSION THREAD

**Discussion by QL9527** [zouyaoji/vue-cesium#235] [Q&A]
当我在使用transfrom:scale实现屏幕自适应时，会导致cesium坐标不准，点击地图上面的实体会获取不到，因为已经错位了。浏览相关文章，目前只有一个方案，将实现的地图组件单独放在一个路由中，再用iframe引入，但是会再次引入项目资源，性能不佳。请问还有更优的方案吗

**Comment by zouyaoji** [comment]
没有相关经验了
